app/assets/javascripts/application.js

```diff
@@ -9,10 +9,10 @@
 //= require clipboard
 //= require_tree .
 
-function handleClick(event, nav, removeNavExpandedClass, addNavExpandedClass) {
+function handleClick(e, nav, removeNavExpandedClass, addNavExpandedClass) {
```

Comment:
What is the reason for this change? I would say that `event` is more explicit (and thus more clear).

Reply:
everywhere else is using "e" so I was going for consistency.   I can revert this if desired.

Reply:
@jvanbaarsen did a force push removing the changes from this file.